src/AltinnCore/Runtime/Startup.cs

```diff
@@ -512,7 +512,7 @@ public void Configure(IApplicationBuilder appBuilder, IHostingEnvironment env)
 
             appBuilder.UseSwaggerUI(c =>
             {
-                c.SwaggerEndpoint("/swagger/v1/swagger.json", "Altinn Apps Runtime API");
+                c.SwaggerEndpoint("v1/swagger.json", "Altinn Apps Runtime API");
```

Comment:
is there a reason for making this change only for this component? Shouldn't the swagger endpoint follow the same structure both for storage and runtime?